src/profiling/meter_manager.cpp

```diff
@@ -142,6 +159,42 @@ void save_to_file(const meter_manager& manager, const std::string& name) {
     }
 }
 
+// Print easy to read report of meters so a stream.
+void print(const meter_manager& manager, std::ostream& o) {
```

Comment:
Agreed, I have improved this based on your suggestions. Now there is an `operator<<`, and the report struct is generated directly by the caller (the minapp in our case), and output from the calling code. The `util::save_to_file` has been removed completely.